tests/Application/config/bundles.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+declare(strict_types=1);
```

Comment:
No OS header found, could you add it globally to all files, on which you've put the strict_types? :)